fastapi-sample/schema.py

```diff
@@ -0,0 +1,108 @@
+from fastapi import FastAPI
+
+app = FastAPI()
+
+# json読み込み準備
+import json
+with open('./sample.json', 'r') as temp:
+    sample_data = json.load(temp)
+
+
+
+# モックを使ってデータ処理以下
+
+
+# 型チェック用の定義
+from pydantic import BaseModel
+from typing import List
+
+class Item(BaseModel):
+    Message: str
+
+class Items(BaseModel):
+    data:List[Item]
+
+# データの取得
+
+@app.get("/items",response_model=Items)
+async def get_messages():
+    """
+        型チェック(Items)をしてデータを返す
+
+        http://localhost:8000/items
+    """
+    return sample_data
+
+
+@app.get("/item/{id}",response_model=Item)
+async def get_item(id:int):
+    """
+        jsonデータからパスパラメーターで要素を指定して返す    
+    
+    
+        http://localhost:8000/item/0
+        http://localhost:8000/item/1
+        http://localhost:8000/item/2
+    """
+
+    return sample_data["data"][id]
+
+
+# データの挿入
+
+@app.post("/item/append",response_model=Item)
+async def create(item:Item):
+    """
+        postをするとsample_dataに追加して送られたものを返す
+
+        curl -X 'POST' \
+        'http://localhost:8000/item/append' \
+        -H 'accept: application/json' \
+        -H 'Content-Type: application/json' \
+        -d '{"Message": "ここに追加したいメッセージを入れる"}'
+
+
+    """
+
+    sample_data["data"].append({"Message":item.Message})
+
+    return {"Message":item.Message}
+
+
+# データの更新
+
+@app.patch("/item/update/{id}",response_model=Item)
```

Comment:
```suggestion
@app.patch("/items/{id}/update",response_model=Item)
```
こちらも同様にこのような設計になるとか思われます!

`````
items ----- index
               |
               ---- items/{item_id} ---- index
                                                  ---- update
                                                  ---- delete 
                                                  ---- append
```